torch/csrc/jit/codegen/cuda/transform_replay.cpp

```diff
@@ -644,24 +644,173 @@ std::pair<TensorDomain*, unsigned int> TransformReplay::replayCasP(
   return replayCasP(consumer, producer, compute_at_axis, root_map);
 }
 
+namespace {
+
+int getMatchedLeafPosWithoutReplay(
```

Comment:
Please add comments on the parameters and the return value.

Reply:
I agree that these functions need a better doc. I will work on it in a new PR.